### DISCUSSION TITLE
Wrong slash for Windows systems

### DISCUSSION THREAD

**Discussion by jczanfona** [aleksey-hoffman/sigma-file-manager#121] [Q&A] [answered]
The slash used is for linux, not for Windows...

I think the trick of changing it in address bar can be confusing...

At lest in panel info would be necessary to copy (`Ctrl + LClick`) with the right one

I'm not sure this approach is going to work.

**Comment by aleksey-hoffman** [comment]
I'm intentionally using only forward `/` slashes throughout the app to unify the experience.

Windows can handle forward `/`  slashes as well as backward `\`  slashes, so it shouldn't be a problem, they will both work in any default Windows app.

Why do you need to use backward  `\`  slashes?

**Reply by jczanfona** [reply]
Windows programs do not recognize this change. 
Without going any further PAINT does not

**Reply by jczanfona** [reply]
Is not possible to open files with that kind of path.

**Reply by jczanfona** [reply]
Nor is that path readable for the command console

**Reply by aleksey-hoffman** [reply]
How are you opening a file path in Paint?

**Reply by aleksey-hoffman** [reply]
Console can read forward slashes as well:

![image](https://user-images.githubusercontent.com/61761672/124935786-a8239400-e00e-11eb-86bb-d9c3fc99a712.png)

 Can you show me an example where it's not working?

**Reply by jczanfona** [reply]
From the menu: File - Open

**Reply by jczanfona** [reply]
dir C:/webs/ci.jpg
El formato del parámetro no es correcto: "webs".

**Reply by aleksey-hoffman** [reply]
You should wrap paths with quotes, e.g. `dir "C:/webs/ci.jpg"`

**Reply by jczanfona** [reply]
Then says the file is not found (without error)

**Reply by aleksey-hoffman** [reply]
Yeah, you're right, for this command it doesn't work. Windows is more inconsistent than I thought.
Ok, I might have to add a settings to use the `\` slashes on Windows.

**Reply by jczanfona** [reply]
Believe me, the best solution is to change it when copying the path from the info panel

**Comment by aleksey-hoffman** [comment, marked as answer]
Fixed in 86695f2aa2629000534141cdeec2b4c959ae912d
